use_cases/eluc/README.md

```diff
@@ -104,14 +103,12 @@ a point in time when the decision had to be made, and the current usage of the l
   - c3fnx
   - pastr
   - range
-  - nonland
 
 Latitude and longitude represent the cell on the grid.  
 Area represents the surface of the cell. Cells close to the equator have a bigger area than cells close to the poles.  
 Year is useful to capture historical decisions: the same cell has been through a lot of land use changes
 over the years.  
-Land usage represents the percentage of the land used by each land type. Note there is a 'nonland' type that represents
-the percentage of the cell that is not land (e.g typically sea, lake, etc.). The land usage types sum up to 100%. 
+Land usage represents the percentage of the land used by each land type. Note that the land usage does not sum to 100% because of area in the cell that is not land (e.g typically sea, lake, etc.).
```

Comment:
I updated this description because it was a little confusing since there isn't actually a 'nonland' column

Reply:
Right, thanks for clarifying that.